### PR TITLE
chore: release

### PR DESCRIPTION
**Pull request by thundergolfer**
Doing via PR because this repo has branch protection on.

### CODE REVIEW

File: blobnet/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "blobnet"
-version = "0.2.10"
+version = "0.2.11"
```
Comment (ekzhang) [inline blobnet/Cargo.toml:3]: Can we push to 0.3? There's a `pub fn` API change

Comment (thundergolfer) [reply blobnet/Cargo.toml:3]: Ahh, yes. I did think about this but discounted the `FileClient::new` change. Had tunnel vision on the `::new_http` method.

Comment (thundergolfer) [reply blobnet/Cargo.toml:3]: `0.3.0` it is.

Comment (thundergolfer) [reply blobnet/Cargo.toml:3]: Thanks for catching that 👍

**Pull request merged**